var/spack/repos/builtin/packages/nextflow/package.py

```diff
@@ -7,23 +7,27 @@
 
 
 class Nextflow(Package):
-    """Data-driven computational pipelines"""
+    """Data-driven computational pipelines."""
 
     homepage = "https://www.nextflow.io"
-    url = "https://github.com/nextflow-io/nextflow/releases/download/v0.24.1/nextflow"
+    url      = "https://github.com/nextflow-io/nextflow/releases/download/v21.04.3/nextflow-21.04.3-all"
 
-    version('20.10.0', sha256='54f76c83cbabe8ec68d6a878dcf921e647284499f4ae917356e594d873cb78dd', expand=False)
-    version('20.07.1', sha256='de4db5747a801af645d9b021c7b36f4a25c3ce1a8fda7705a5f37e8f9357443a', expand=False)
-    version('0.25.6', sha256='9498806596c96ba87396194fa6f1d7d1cdb739990f83e7e89d1d055366c5a943', expand=False)
-    version('0.24.1', sha256='0bfde5335b385e3cff99bf4aab619e583de5dc0849767240f675037a2e7c1d83', expand=False)
-    version('0.23.3', sha256='ffe1c314962ff97ebf47b0567883e152522acfbf6fd5800200b1a7a0ca2896d2', expand=False)
-    version('0.21.0', sha256='076089079479da0d91fe1ad7aad06816164ecbcf17f73c55e795b1db8462b28d', expand=False)
-    version('0.20.1', sha256='02635f3371f76a10e12f7366508c90bacf532ab7c23ae03c895317a150a39bd4', expand=False)
-    version('0.17.3', sha256='05563ee1474fbef22f65fa3080792dcb08d218dd1b1561c517ebff4346559dbe', expand=False)
```

Comment:
Instead of removing old versions, the spack policy (for now) is to leave them in and add the `, deprecated=True` keyword argument. This allows older specs/environments to be reproduced on newer versions of spack if necessary.

Reply:
I can do that, but here the checksums will change as well